site/docs/design_tokens/colour.mdx

```diff
@@ -87,10 +89,58 @@ Status colours are reserved exclusively for indicating valid, invalid or informa
 | <ColorBox color="var(--color-black)"/> | <ColorBox color="var(--color-black-90)"/> | <ColorBox color="var(--color-black-80)"/> | <ColorBox color="var(--color-black-70)"/> | <ColorBox color="var(--color-black-60)"/> | <ColorBox color="var(--color-black-50)"/> | <ColorBox color="var(--color-black-40)"/> | <ColorBox color="var(--color-black-30)"/> | <ColorBox color="var(--color-black-20)"/> | <ColorBox color="var(--color-black-10)"/> | <ColorBox color="var(--color-black-5)"/> | <ColorBox color="var(--color-white)" /> |
 
 
+## Accessible colour combinations
+<p>Here you can find some recommended colour combinations for text and other informative elements. The examples are sorted in order of their contrast ratio. For more examples of accessibile colour combinations, see the <Link href="https://share.goabstract.com/28352b8b-9c51-4346-b249-ff062c5da561" external>Colour accessibility examples - Abstract collection</Link>.</p>
+
+- **AA** and **AAA** means you can safely use the color combination on all text sizes
+- **AA for large text** means the text size should be at least 18 pt for regular weight or 14 pt for bold weight
+- **Black 90** is the default text colour in HDS
+
+### Brand colour combinations
+
```

Comment:
Lisäisin tähän väliin jonkun lyhyen lauseen, joka selittää mitä seuraavassa taulukossa on. Vähän niin kuin alemmassakin taulukossa on selite.

Reply:
Tämän selite oli tavallaan tuo alun "Here you can find some recommended colour combinations…" -teksti. Lisäsin selkeyden vuoksi väliotsikon tälle ja grayscalelle. Ei ehkä kannata toistaa alun löpinöitä sekä tässä että grayscalessa erikseen.